portfolio/src/main/webapp/script.js

```diff
@@ -66,3 +67,9 @@ async function getUserImages(){
       imagesContainer.appendChild(imageNode);
     });
 }
+
+function createMap() {
+  const map = new google.maps.Map(
+      document.getElementById('map'),
+      {center: {lat: 37.422, lng: -122.084}, zoom: 16});
```

Comment:
nit.  that lat/lng is a magic number https://en.wikipedia.org/wiki/Magic_number_(programming). document what it is or assign to a name constant(s).